test/scale-infra/INSTRUCTIONS.md

```diff
@@ -0,0 +1,100 @@
+### Examples of Individual Steps
+As an alternative to the quick-start deployment bootstrapping instructions, you can also run the individual steps starting from a local directory containing the git repo as follows:
+
+1. Create the KubeStellar core infra on AWS. The infrastructure includes a VPC, security groups, EC2 instances, etc.
+
+    ```bash
+    cd test/scale-infra
+    ansible-playbook deploy_vpc_core.yaml -e "region=us-east-2 name=<aws-iam-user>"
+    ansible-playbook create-ec2.yaml -e "cluster_name=core region=us-east-2 vpc_name=name=<vpc_name> aws_key_name=mykey  num_masters=1 num_workers=2 instance_type=t2.xlarge arch=x86_64 ec2_image='<your-aws-ami>'"
+    ```
+
+    Use the flag `--vpc-name` to specify the name for the [AWS virtual private cloud](https://docs.aws.amazon.com/vpc/latest/userguide/what-is-amazon-vpc.html) to deploy your infrastructure in a logically isolated virtual network: *We highly advise utilizing your name or the AWS IAM user ID as the identifier for your VPC*. Furthermore, use the flag `--aws-ami` to specify the [Amazon machine image ID](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html), keeping in mind that it is region-specific.
+
+    Upon completion of the script's execution, an Ansible inventory file containing the IP addresses of the master and worker nodes will be generated in the present directory at `.data/hosts_core`.
+
+2. Deploy Kubernetes clusters:
+
+    ```bash
+    ansible-playbook -i .data/hosts_core deploy-masters.yaml --ssh-common-args='-o StrictHostKeyChecking=no'
+    ansible-playbook -i .data/hosts_core deploy-workers.yaml --ssh-common-args='-o StrictHostKeyChecking=no'
+    ```
+
+3. Deploy KubeStellar in the hosting cluster:
+
+    ```bash
+    ansible-playbook -i .data/hosts_core deploy_ks_core.yaml --ssh-common-args='-o StrictHostKeyChecking=no' -e 'ks_release=0.25.1'
```

Comment:
Shouldn't this use `{{ config.ks_latest_release }}` instead of `0.25.1`?

Reply:
Where is variable `{{ config.ks_latest_release }}`  coming from? My intention is to make the KS release parametrize, not necessarily only to the latest version.

Reply:
See https://docs.kubestellar.io/release-0.26.0/contribution-guidelines/operations/document-management/#global-variables

If you want to maintain documentation that covers multiple releases, then the document needs to identify the releases that it works with.

Reply:
I still see only a literal release string.

Reply:
ok, I updated it with the suggested variable: `{{ config.ks_latest_release }}`

Reply:
OK.
However, now I see that maybe I was asking for the wrong thing. This document is _not_ rendered to the website, right? The notation `{{ config.ks_latest_release }}` only works in that rendering process. If this document is NOT rendered to the website then you need some other way to indicate the release to use.

Also, if I understand correctly, these instructions are expected to work with a range of releases. Would it be accurate to say that the current vesion of this PR will work with any release that is `>= 0.25.1` ? If so then a solution might look like telling the reader to set an evironment variable named KS_RELEASE to `0.25.1` or later.

Same considerations apply to `test/scale-infra/README.md`.

Reply:
Fixed! I added the following statement in both `INSTRUCTIONS.md` and `README.md` as suggested above:
```
Set var KS_RELEASE to release 0.25.1 or later.
```